### DISCUSSION TITLE
FORGE--zksync issues.

### DISCUSSION THREAD

**Discussion by DevinBeu** [Cyfrin/foundry-full-course-cu#3019] [General]
![glibc](https://github.com/user-attachments/assets/a71a5ac5-2ee8-4808-b460-5d88075472ae)
![zksync](https://github.com/user-attachments/assets/77ba9b8a-579b-4216-8801-98ec7234a46b)
It looks like the GLIBC package is a central package for my OS. i am worried trying to manually update it might break some shit? Does anyone have a pointer in the right direction?(or a direction)

**Comment by EngrPips** [comment]
Hello @DevinBeu, Please read this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2052#discussioncomment-11349762 It should help you resolve your issue.

**Comment by irjan0154** [comment]
Tnx! Very good)

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.
